ext/native-decls/GetEntityFromStateBagName.md

```diff
@@ -0,0 +1,15 @@
+---
+ns: CFX
+apiset: shared 
+---
+## GET_ENTITY_FROM_STATE_BAG_NAME
+
+```c
+Entity GET_ENTITY_FROM_STATE_BAG_NAME(char* bagName);
+```
+
+## Parameters
+* **bagName**: The bag name
+
+## Return value
+The entity handle or 0 if it didn't exist
```

Comment:
`An entity handle, or 0 if the state bag name did not refer to an entity, or the entity does not exist.`